lua/bufferline/commands.lua

```diff
@@ -46,10 +46,14 @@ local function open_element(id)
 end
 
 ---@param id number
-local function delete_element(id)
+local function delete_element(id, force)
   if config:is_tabline() then
     vim.cmd("tabclose " .. id)
   else
+		local is_modified = api.nvim_buf_get_option(id, 'modified')
+		if is_modified and not force then
```

Comment:
The indentation here is off, can you align this with other code at this level e.g line 51. Also
```suggestion
if vim.bo[id].modified and not force then
```

Reply:
@yangyuanxx I don't think you saw the review comment about changing the `is_modified` variable to just use the `vim.bo[id]` syntax